entrypoint/airgap/systemdefaultregistry_test.go

```diff
@@ -0,0 +1,53 @@
+//go:build systemdefaultregistry
+
+package airgap
+
+import (
+	"fmt"
+
+	"github.com/rancher/distros-test-framework/pkg/assert"
+	"github.com/rancher/distros-test-framework/pkg/testcase"
+	"github.com/rancher/distros-test-framework/shared"
+
+	. "github.com/onsi/ginkgo/v2"
+)
+
+var _ = Describe("Test Airgap Cluster with System Default Registry:", Ordered, func() {
+	It("Creates bastion and private nodes", func() {
+		testcase.TestBuildAirgapCluster(cluster)
+	})
+
+	It("Installs and validates product on private nodes:", func() {
+		testcase.TestSystemDefaultRegistry(cluster, flags)
+	})
+
+	It("Validates Nodes", func() {
+		testcase.TestAirgapClusterNodeStatus(
+			cluster,
+			assert.NodeAssertReadyStatus(),
+			nil,
+		)
+	})
+
+	It("Validates Pods", func() {
+		testcase.TestAirgapClusterPodStatus(
+			cluster,
+			assert.PodAssertRestart(),
+			assert.PodAssertReady())
+	})
+
+	AfterAll(func() {
+		shared.DisplayAirgapClusterDetails(cluster)
+	})
+
+	// TODO: Validate deployment, eg: cluster-ip
```

Comment:
this will be another PR?

Reply:
Yes once we figure out how to do it

Reply:
got it ok...will finish review tm

Reply:
what exactly u want to validate here?  cause my current PR maybe help you on that

Reply:
I want to be able to deploy workload and validate cluster-ip, pod-to-pod communication etc... nothing outlined as of now

Reply:
ow ok, got it